python/extensions/pip.bzl

```
@@ -316,9 +437,55 @@ Each `pip.parse()` call configures a particular Python version. Multiple calls
 can be made to configure different Python versions, and will be grouped by
 the `hub_name` argument. This allows the same logical name, e.g. `@pip//numpy`
 to automatically resolve to different, Python version-specific, libraries.
+
+pip.whl_mods:
+This tag class is used to create different wheel modification JSON files.  These files
+contain directives that are used by the wheel_installer.py during the creation of 
+wheels.
```

Just trimming some low-level details users don't need to know about.

```suggestion
This tag class is used to help create JSON files to describe modifications to
the BUILD files for wheels.
```